style.css

```diff
@@ -135,11 +161,285 @@ body {
+    margin-bottom: 33px;
+  }
+
+  .menu__item::after {
+    visibility: hidden;
+  }
+
+  .menu__item_active::after {
+    visibility: hidden;
+  }
+
+  .main {
+    text-align: center;
+    padding: 80% 0 0;
+    background-size: 95%;
+    background-position: top center;
+  }
+
+  .main__title {
+    margin: 0 0 33px 6px;
+    padding-top: 25px;
+    font: bold 30px/39px "pt_serif", arial, helvetica, sans-serif;
+    text-align: center;
+  }
+
+  .main__subtitle {
+    margin: 0 25px;
+    padding-bottom: 25px;
+    font: 500 16px/19px "Roboto", arial, helvetica, sans-serif;
+    text-align: center;
+  }
+
+  .main__button {
+    padding: 14px 21px;
+    margin: 0 auto 25px;
+    font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+  }
+
+  #menu__toggle {
+    opacity: 0;
+  }
+
+  .menu__btn {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    left: 326px;
+    top: 37px;
+    width: 25px;
+    height: 3px;
+    cursor: pointer;
+    z-index: 1;
+  }
+
+  .menu__btn > span,
+  .menu__btn > span::before,
+  .menu__btn > span::after {
+    display: block;
+    position: absolute;
+    width: 100%;
+    height: 3px;
+    background-color: #0056ec;
+  }
+
+  .menu__btn > span::before {
+    content: '';
+    top: -8px;
+  }
+
+  .menu__btn > span::after {
+    content: '';
+    top: 8px;
+  }
+
+  .menu__box {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    width: 375px;
+    height: 100%;
+    margin: 0;
+    padding: 86px 0;
+    text-align: center;
+    background-color: #fff;
+  }
+
+  .menu__item {
+    display: block;
+    font-family: "Roboto", arial, helvetica, sans-serif;
+    font-weight: 500;
+    top: 79px;
+    padding: 15px;
+    margin: 10px;
+    font-size: 29px;
+  }
+
+  .menu__item:nth-child(1) {
+    padding: 18px;
+    margin-right: 9px;
+  }
+
+  .menu__item:nth-child(2) {
+    padding: 10px;
+    margin-right: 12px;
+  }
+
+  .menu__item:nth-child(3) {
+    padding: 19px 0 3px 0;
+    margin-right: 14px;
+  }
+
+  .menu__item:nth-child(4) {
+    padding: 27px;
+    margin-right: 18px;
+  }
+
+  .menu__item:nth-child(5) {
+    padding: 2px;
+    margin-right: 17px;
+  }
+
+  #menu__toggle:checked ~ .menu__btn > span {
+    transform: rotate(45deg);
+  }
+
+  #menu__toggle:checked ~ .menu__btn > span::before {
+    top: 0;
+    transform: rotate(0);
+  }
+
+  #menu__toggle:checked ~ .menu__btn > span::after {
+    top: 0;
+    transform: rotate(90deg);
+  }
+
+  #menu__toggle:checked ~ .menu__box {
+    visibility: visible;
+    left: 0;
+  }
+}
+
+@media (min-width: 769px) and (max-width: 1024px) {
+  .wrapper {
+    width: 91%;
+  }
+
+  .header {
+    margin-bottom: 66px;
+  }
+
+  .menu__item {
+    font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+  }
+
+  .menu__item:nth-child(1) {
+    margin-right: 21px;
+  }
+
+  .menu__item:nth-child(2) {
+    margin-right: 20px;
+  }
+
+   .menu__item:nth-child(3) {
+     margin-right: 23px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6297664)